utils/copy-env-vars.sh

```diff
@@ -0,0 +1,9 @@
+folders=("dtfs-central-api" "external-api" "gef-ui" "portal" "portal-api" "secrets" "trade-finance-manager-api" "trade-finance-manager-ui" "utils/mock-data-loader")
```

Comment:
For running cypress locally we might need .env in these folders too:
* e2e-tests/tfm
* e2e-tests/gef
* e2e-tests/portal
* e2e-tests/ukef

Reply:
I don't have any of these set up and can run e2e tests fine? Does anyone else have/not have these please

Reply:
I run e2e test in this way:
```
cd e2e-tests;
npx cypress open --project ./tfm;
```
What is your way?

Reply:
.env in e2e-tests/tfm is required for my way

Reply:
I do it the same... 
cd into e2e and `npx cypress open --project ./gef`...

I don't have the env vars in those folders. Odd. I'm happy to add if it helps you though

Reply:
ah not ran recently with sql changes. adding

Reply:
Should not need `.env` inside all `e2e` just inside `e2e-tests` will suffice.
Please change:
* Remove `secrets`
* Update to `utils`
* Add `azure-functions/acbs-function`